README.md

```diff
@@ -428,7 +470,33 @@ For more customization, creating a [HeaderProvider](https://sap.github.io/cloud-
 export AICORE_SERVICE_KEY='{   "serviceurls": {     "AI_API_URL": ...'
 ```
 
-### Run the Spring Boot application
+### Define and use a Destination
+
+* Lookup service-key credentials as explained in the previous step for `AICORE_SERVICE_KEY`.
+
+* <details><summary>Define a new destination in the SAP BTP Destination Service using the service-key credentials</summary>
+  
+  * (Destinations can be added on subaccount level and on service instance level.)
+  * (The URL field requires an additional path segment: `/v2`)
+  * **Name**: _my-aicore_
+  * **Type**: HTTP
+  * **URL**: _[serviceurls.AI_API_URL]/v2_
+  * **Proxy-Type**: Internet
+  * **Authentication**: _Oauth2ClientCredentials_
+  * **Client ID**: _[clientid]_
+  * **Client Secret**: _[clientsecret]_
+  * **Token Service URL Type**: Dedicated
+  * **Token Service URL**: _[url]_
+  
+  </details>
+
+* At application runtime the following can be executed:
+  ```java
+  Destination destination = DestinationAccessor.getDestination("my-aicore");
```

Comment:
Does the customer need to add the Cloud SDK dependency, or can you use `DestinationAccessor ` with the transitive dependency?